### ISSUE TITLE
GaiaTransform

### ISSUE THREAD

**Issue by wxy656** [MTG/essentia#453]
when i use GaiaTransform to predict highlevel descriptors as below：
           predicte=est.GaiaTransform(history = 'essentia/svm_models/%s.history' % type_)(est.YamlInput(filename = "audio.json")())

it works successufully ,but my problem is if i can get the Parameter (equal to "est.YamlInput(filename = "audio.json")()")  straightly in python rather than load a file created previously .  
I have tried  the  Extractor  function ,but the returned pool instance from it  can not be used as  GaiaTransform's  Parameter.  An error is that  "lowlevel" field  in audio.json file is spelled as "lowLevel" in Extractor  function and it loss some fields(ex: lowlevel.barkbands_crest)  compare to audio,json file

Is there a function in python that  can get the pool instance  for GaiaTransform?

**Comment by dbogdanov** [comment]
The pool produced by Extractor is incompatible with the pre-trained models. 

You are supposed to create the required pool. In the case you are using our standard models, you'll have to reconstruct exactly the same pool as the one produced by the [music extractor](http://essentia.upf.edu/documentation/streaming_extractor_music.html). We might consider wrapping this extractor into Python too, but not at this moment. 

If you are using your own trained models, you can produces the pool the same way as you have computed it for training your instances.

**Comment by snape6666** [comment]
I tried to reconstruct the pool from music extractor... but i didnt made it...

do you have a python code with the correct output for the pretrained svm's?
